src/main/webapp/resources/js/dv_rebind_bootstrap_ui.js

```diff
@@ -213,7 +213,6 @@ function contentTruncate(truncSelector, truncMoreBtn, truncMoreTip, truncLessBtn
  */
 function checksumTruncate(){
     $('span.checksum-truncate').each(function () {
-        $(this).toggleClass('sr-only').toggleClass('visisble');
```

Comment:
I know @qqmyers mentioned this in the issue but it's so weird to see this misspelling: `visisble`. I wonder what is or was going on here? Anyway, I guess it's going away. Looks like it was added in 89023ba.